### PR TITLE
simplified findMobTimerProcessIdsLinuxAndDarwin and fix mac CI build

### PR DESCRIPTION
**Pull request by hollesse**

### CODE REVIEW

**Comment by gregorriegler** [comment]
LGTM. The mac build fails, also on the main branch. Not related to this change however.

**Comment by hollesse** [comment]
@gregorriegler the mac build is working now :)

File: mob_test.go [hunk 1]
```diff
@@ -1508,13 +1507,14 @@ func TestHelpRequested(t *testing.T) {
 }
 
 func TestAbortTimerIfNewTimerIsStarted(t *testing.T) {
-	_, configuration := setup(t)
+	output, configuration := setup(t)
+	say.TurnOnDebugging()
```
Comment (gregorriegler) [inline mob_test.go:1511]: wir haben hier noch debugging an

File: mob_test.go [hunk 2]
```diff
@@ -1528,35 +1528,41 @@ func assertNoTimerProcess(t *testing.T) {
 }
 
 func TestAbortBreakTimerIfNewBreakTimerIsStarted(t *testing.T) {
-	_, configuration := setup(t)
+	output, configuration := setup(t)
+	say.TurnOnDebugging()
```
Comment (gregorriegler) [inline mob_test.go:1532]: auch hier

File: mob_test.go [hunk 3]
```diff
@@ -1528,35 +1528,41 @@ func assertNoTimerProcess(t *testing.T) {
...
 
+	assertOutputContains(t, output, "Killed mob timer with PID")
 	assertSingleTimerProcess(t)
 	abortRunningTimers()
 }
 
 func TestAbortTimerIfMobNext(t *testing.T) {
-	_, configuration := setup(t)
+	output, configuration := setup(t)
+	say.TurnOnDebugging()
```
Comment (gregorriegler) [inline mob_test.go:1545]: debugging

File: mob_test.go [hunk 4]
```diff
@@ -1528,35 +1528,41 @@ func assertNoTimerProcess(t *testing.T) {
...
 	next(configuration)
 
+	assertOutputContains(t, output, "Killed mob timer with PID")
 	assertNoTimerProcess(t)
 }
 
 func TestAbortTimerIfMobDone(t *testing.T) {
-	_, configuration := setup(t)
+	output, configuration := setup(t)
+	say.TurnOnDebugging()
```
Comment (gregorriegler) [inline mob_test.go:1558]: debugging

**Review by gregorriegler — APPROVED** [review]

**Comment by gregorriegler** [comment]
ok, now I understand that it relies on the debugger. Thats weird. Let me think about how we can fix that.

**Comment by hollesse** [comment]
Correct at the moment we just have debug logs for it. Maybe it makes sense to have a info log that tells the user that the timer was overwritten or aborted. But I'm not sure if the user really needs that info or if this would be noise as this is behavior the new user would expect anyways

**Comment by gregorriegler** [comment]
I made the change to just not check that. We know that the Process is there, so we know it must have been removed. No need to check the logs. And now we don't change the log level for other tests. If we still want to test whether it was invoked, this should in my opinion be done on a more granular level, where the kill function is injected and replaced with a spy.
If the test are green, I'm fine with merging this.

**Comment by hollesse** [comment]
Ah correct I missed that this is the activated for all tests. Then I am fine with removing this assert. Just merge it if the tests are green

**Pull request merged**